tools/dev/loki-boltdb-storage-s3/docker-compose.yml

```diff
@@ -5,6 +5,7 @@ services:
       driver: loki-compose
       options:
         loki-url: "http://localhost:8001/loki/api/v1/push"
+        loki-retries: "0"
```

Comment:
Does this not also have the side-effect of not retrying any batches if the loki server goes away temporarily? I'd imagine that could lead to confusion when using this for development.

Reply:
Sure. Can we add a comment here describing the trade-off, and a caution about making it a non-zero value?